### ISSUE TITLE
Possible bug in the computation of detection metrics for scikit-learn baselines

### ISSUE THREAD

**Issue by vdumoulin** [earthspecies/beans#3]
On detection tasks, and for `sklearn` models, the [`model.predict`](https://github.com/earthspecies/beans/blob/main/scripts/evaluate.py#L51) call returns binary predictions for each class rather than floating point scores, which means that the `MeanAveragePrecision` metric ranks lists of "scores" in {0, 1}. I believe the `model.predict_proba` method should be used instead for detection tasks, is that correct?
